charts/security-apps/values.yaml

```
@@ -18,8 +18,30 @@ dex:
   # @default -- [upstream values](https://github.com/dexidp/helm-charts/tree/master/charts/dex/values.yaml)
   values: {}
 
-# gangway -- [gangway](https://github.com/heptiolabs/gangway/) ([example](./examples/gangway.yaml))
+# dexK8sAuthenticator -- [dex-k8s-authenticator](https://github.com/mintel/dex-k8s-authenticator) ([example](./examples/dex-k8s-authenticator.yaml))
 # @default -- -
+dexK8sAuthenticator:
+  # dexK8sAuthenticator.enabled -- Enable dex-k8s-authenticator
+  enabled: false
+  name: dex-k8s-authenticator
+  destination:
+    # dexK8sAuthenticator.destination.namespace -- Namespace
+    namespace: "infra-dex-k8s-authenticator"
+  # dexK8sAuthenticator.repoURL -- Repo URL
+  # @default -- [repo](https://github.com/mintel/dex-k8s-authenticator.git)
+  repoURL: "https://github.com/mintel/dex-k8s-authenticator.git"
+  # dexK8sAuthenticator.repoPath -- Repo Path
+  repoPath: "charts/dex-k8s-authenticator"
+  # dexK8sAuthenticator.chart -- Chart
+  chart: "dexK8sAuthenticator"
+  # dexK8sAuthenticator.targetRevision -- [dex-k8s-authenticator Helm chart](https://github.com/mintel/dex-k8s-authenticator/tree/master/charts/dex-k8s-authenticator/) version
+  targetRevision: "v1.4.0"
```

we should really look for a real repo so we can wildcard some:
```suggestion
  targetRevision: "1.4.*"
```

otherwise this feels very hacky

---

Upstream helm chart repo never finished
FFI: https://github.com/mintel/dex-k8s-authenticator/issues/185